randovania/games/dread/json_data/Cataris.txt

```diff
@@ -3389,29 +3585,29 @@ Extra - asset_id: collision_camera_051
   * Extra - left_shield_def: None
   * Extra - right_shield_entity: {EMPTY}
   * Extra - right_shield_def: None
-  > Door to Diffusion Beam Room
-      Morph Ball and After Cataris - db_dif_mg_001
   > Teleporter to Dairon - Teleport to Cataris
-      Trivial
+      Varia Suit or Heat Damage ≥ 30
   > Tile Group (POWERBEAM) 2
-      Trivial
+      Varia Suit or Heat Damage ≥ 10
   > Dock to Kraid Eyedoor Room
-      Morph Ball and After Cataris - db_reg_mg_021
+      All of the following:
+          After Cataris - db_reg_mg_021
+          Varia Suit or Heat Damage ≥ 100
   > Event - Wave Beam Blob (db_reg_mg_021)
       Wave Beam
 
 > Event - Missile Tank Blob (breakablemag009); Heals? False
   * Event Cataris - breakablemag009
   * Extra - actor_name: breakablemag009
   * Extra - actor_def: actordef:actors/props/breakablemag009/charclasses/breakablemag009.bmsad
-  > Teleporter to Dairon - Teleport to Cataris
-      Trivial
+  > Ledge above Teleport
+      Varia Suit or Heat Damage ≥ 50
 
-> Event - Diffusion Tutorial Blob (db_dif_mg_001); Heals? False
+> Event - Diffusion Tutorial Blob from below (db_dif_mg_001); Heals? False
   * Event Cataris - db_dif_mg_001
   * Extra - actor_name: db_dif_mg_001
   * Extra - actor_def: actordef:actors/props/db_dif_mg_001/charclasses/db_dif_mg_001.bmsad
-  > Door to Diffusion Beam Room
+  > Dock to Diffusion Beam Room
```

Comment:
This room (Teleport to Dairon) has a few logic errors (I think?):
- The wave beam blob event is not properly set up
- The tutorial blob from above is innaccessible from the rightmost door and from the area below
- The dock to Kraid Eyedoor room is trivially accessible

Reply:
I rechecked in-game, and my second point is wrong because there is should be only one blob, not two.

Reply:
- The wave beam blob is already hooked up to db_reg_mg_021. What else is missing from it?
- I reworked the logic of the dock to Kraid Eyedoor Room, so heat logic is easier to segregate